vita3k/interface.cpp

```diff
@@ -648,6 +649,9 @@ bool handle_events(EmuEnvState &emuenv, GuiState &gui) {
         }
     };
 
+    for (int i = 0; i < IM_ARRAYSIZE(io.NavInputs); i++)
+        io.NavInputs[i] = 0.0f;
```

Comment:
NavInput has the following comment:
[LEGACY] Since 1.88, NavInputs[] was removed. Backends from 1.60 to 1.86 won't build. Feed gamepad inputs via io.AddKeyEvent() and ImGuiKey_GamepadXXX enums.

Are you sure about this? I am fine with using internal ImGui functions but we should at least not use deprecated/about to be removed ones.